src/meta/model/src/fragment.rs

```diff
@@ -30,6 +30,7 @@ pub struct Model {
     /// Specifically, `Merge` nodes' `upstream_actor_id` will be filled. (See `compose_fragment`)
     pub stream_node: StreamNode,
     pub state_table_ids: I32Array,
+    #[deprecated]
     pub upstream_fragment_id: I32Array,
```

Comment:
Are we going to remove this field?

Reply:
May keep it for a release and remove it in the next release.